bin/dfx-ii-create-dummy-anchor

```diff
@@ -28,18 +28,15 @@ clean_up() {
 
 trap clean_up EXIT
 
-DECLARATIONS_DIR="$(realpath "$SOURCE_DIR/../declarations")"
-II_DID_FILE="$DECLARATIONS_DIR/internet_identity.did"
```

Comment:
This is not where the upstream `dfx nns import` places the II .did file.  If the .did file location is unspecified, it will be picked up from `dfx.json` so let's delete this variable.  Perhaps in future all we will need is the II canister ID; we can get the .did from the deployed canister metadata, if we don't have it already.